components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Deploy/IoTHubDiscovererDeployment.cs

```diff
@@ -95,15 +95,7 @@ public sealed class IoTHubDiscovererDeployment : IHostProcess {
                     Cmd = new[] {
                         "PkiRootPath=/mount/pki"
                     },
-                    NetworkingConfig = new {
```

Comment:
Can we break this file into a separate PR?  is the discovery module still able to enumerate the subnets of the network adapters?  Or does this require now to also specify subnet mask?

Reply:
sure, I will move this to a separate PR. 
with this change the user needs to specify the subnet mask, in a same manner as it needs to be done for Windows containers. Do you think this is a blocker?

Reply:
Yes, let's understand the root cause a bit more before we make the other change.  We can deal with it in the other PR.